website/docs/r/sentinel_log_analystic_workspace_onboard.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Sentinel"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_sentinel_log_analytics_workspace_onboard"
```

Comment:
given the description below uses onboarding
```suggestion
page_title: "Azure Resource Manager: azurerm_sentinel_log_analytics_workspace_onboarding"
```

as "onboard" doesn't seem like the right name?